### ISSUE TITLE
Implement missing systems in UN poster

### ISSUE THREAD

**Issue by ronaldtse** [interscript/interscript#488]
Currently, 23 of systems are missing as following

- Assamese [UN 1972, II/11; 1977, III/12]
- Bulgarian [UN 1977, III/10]
- Burmese [BGN/PCGN 1970]
- Dzongkha [national 1997]
- Gujarati [UN 1972, II/11; 1977, III/12]
- Kazakh [BGN/PCGN 1979]
- Khmer [UN 1972, II/10]
- Kirghiz [BGN/PCGN 1979]
- Lao [national 1966]
- Macedonian Cyrillic [UN 1977, III/11]
- Malayalam [UN 1972, II/11; 1977, III/12]
- Maldivian [national 1987]
- Persian [UN 1967, I/13]
- Punjabi [UN 1972, II/11; 1977, III/12]
- Serbian [UN 1977, III/11; 2002, VIII/14]
- Sinhalese [provisional 1972]
- Tajik [BGN/PCGN 1994]
- Tamil [UN 1972, II/11; 1977, III/12]
- Telugu [UN 1972, II/11; 1977, III/12]
- Thai [UN 1967, I/14] 
- Tigrinya [BGN/PCGN 2007]
- Ukrainian [national 2007]
- Urdu [UN 1972, II/11; 1977, III/12]

_Originally posted by @webdev778 in https://github.com/interscript/interscript.com/pull/10#issuecomment-700622785_

**Comment by ronaldtse** [comment]
@andrew2net @CAMOBAP can you help with these?
* Bulgarian [UN 1977, III/10]
* Macedonian Cyrillic [UN 1977, III/11]
* Serbian [UN 1977, III/11; 2002, VIII/14]
* Ukrainian [national 2007]
* Kazakh [BGN/PCGN 1979]
* Tajik [BGN/PCGN 1994]
* Kirghiz [BGN/PCGN 1979]

@AhMohsen46 can you help with these?
* Persian [UN 1967, I/13]
* Urdu [UN 1972, II/11; 1977, III/12]

@bilashsaha can you help with these?
* Assamese [UN 1972, II/11; 1977, III/12]
* Gujarati [UN 1972, II/11; 1977, III/12]
* Punjabi [UN 1972, II/11; 1977, III/12]
* Tamil [UN 1972, II/11; 1977, III/12]
* Sinhalese [provisional 1972]
* Malayalam [UN 1972, II/11; 1977, III/12]

@tsega can you handle this?
* Tigrinya [BGN/PCGN 2007]

Other (and abugida) scripts not yet handled (If anyone can handle them please announce yourself):
* Telugu [UN 1972, II/11; 1977, III/12]
* Thai [UN 1967, I/14]
* Khmer [UN 1972, II/10]
* Assamese [UN 1972, II/11; 1977, III/12]
* Burmese [BGN/PCGN 1970]
* Lao [national 1966]
* Dzongkha [national 1997]
* Maldivian [national 1987]

Before starting a new system please file a new issue for that system with the link to its spec. Thanks!

**Comment by AhMohsen46** [comment]
@ronaldtse yes I think I can, but is there a link to a doc for persian?

I found this on google, not sure if it helps, but it looks good to me
http://www.eki.ee/wgrs/v2_2/rom1_fa.htm

**Comment by tsega** [comment]
@ronaldtse Tigrinya and Amharic are very close to each other. I can take that on.

**Comment by ronaldtse** [comment]
Thanks @AhMohsen46 and @tsega !

**Comment by ronaldtse** [comment]
Ping @bilashsaha on these:

Assamese [UN 1972, II/11; 1977, III/12] (we have ALALC's system done)
Gujarati [UN 1972, II/11; 1977, III/12]
Punjabi [UN 1972, II/11; 1977, III/12]
Tamil [UN 1972, II/11; 1977, III/12] (ALALC in #160 , DIN in #456 )
Sinhalese [provisional 1972] (ALALC in #153 )
Malayalam [UN 1972, II/11; 1977, III/12] (ALALC in #131 )

**Comment by bilashsaha** [comment]
@ronaldtse I have made issues to add the missing systems: 
Gujarati [UN 1972, II/11; 1977, III/12] (ALALC in https://github.com/interscript/interscript/issues/538 and https://github.com/interscript/interscript/issues/539)
Punjabi [UN 1972, II/11; 1977, III/12] (ALALC in https://github.com/interscript/interscript/issues/537)

**Comment by ronaldtse** [comment]
Thanks @bilashsaha !

**Comment by bilashsaha** [comment]
For Missing systems

Sinhalese [provisional 1972] (ALALC version(2011) in https://github.com/interscript/interscript/issues/551 )
Malayalam [UN 1972, II/11; 1977, III/12] (ALALC version(2012) in https://github.com/interscript/interscript/issues/551 )
